lib/benchee/system.ex

```diff
@@ -108,29 +109,29 @@ defmodule Benchee.System do
     parse_memory_for(:Linux, system_cmd("cat", ["/proc/meminfo"]))
   end
 
-  @kilobyte_to_gigabyte 1024 * 1024
-  @byte_to_gigabyte 1024 * @kilobyte_to_gigabyte
-
   defp parse_memory_for(_, "N/A"), do: "N/A"
   defp parse_memory_for(:Windows, raw_output) do
     [memory] = Regex.run(~r/\d+/, raw_output)
     {memory, _} = Integer.parse(memory)
-    format_memory(memory, @byte_to_gigabyte)
+    format_memory(memory)
   end
   defp parse_memory_for(:macOS, raw_output) do
     {memory, _} = Integer.parse(raw_output)
-    format_memory(memory, @byte_to_gigabyte)
+    format_memory(memory)
   end
   defp parse_memory_for(:Linux, raw_output) do
     ["MemTotal:" <> memory] = Regex.run(~r/MemTotal.*kB/, raw_output)
     {memory, _} = memory
                   |> String.trim()
                   |> String.trim_trailing(" kB")
                   |> Integer.parse
-    format_memory(memory, @kilobyte_to_gigabyte)
+    format_memory(memory * 1024)
```

Comment:
This impromptu conversion isn't ideal imo. I think it'd be better if we explicitly took it as kilobytes and converted it to bytes and then passed it into th format_memory function